LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp

```diff
@@ -0,0 +1,28 @@
+class Solution {
+public:
+    bool isAnagram(string s, string t) {
+
+if(s.size()!= t.size()){
```

Comment:
The format of your code not good.
Like these line,
```cpp
    bool isAnagram(string s, string t) {

if(s.size()!= t.size()){
    return false;
}
```
correct format will be: 
```cpp
    bool isAnagram(string s, string t) {

    if (s.size() != t.size()) {
        return false;
    }
```

Reply:
Sorry my bad, I solved it on Leetcode and pasted the code from leetcode's IDE, I have reformatted and added indentations to make my code consistent and easy to read, kindly review the same and let me know if any further changes are required